### PR TITLE
chore: update AppDynamics installation guide

### PR DESCRIPTION
**Pull request by tzssangglass**
### Description

 
Refine the AppDynamics plugin documentation with updated installation instructions, distinguishing between recommended practices for Kong version 3.0.0.0 and later versus alternative approaches for older versions or unique environments.

[FTI-5652](https://konghq.atlassian.net/browse/FTI-5652)

### Testing instructions

Preview link: 

### Checklist 

- [ ] Review label added 
- [x] PR pointed to correct branch (`main` for immediate publishing, or a release branch: e.g. `release/gateway-3.2`, `release/deck-1.17`)

[FTI-5652]: https://konghq.atlassian.net/browse/FTI-5652?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by tzssangglass** [comment]
cc @fffonion @windmgc

File: app/_hub/kong-inc/app-dynamics/overview/_index.md [hunk 1]
```diff
@@ -28,12 +28,21 @@ not distributed with {{site.base_gateway}}. AppDynamics
...
-[system's shared library loader](https://tldp.org/HOWTO/Program-Library-HOWTO/shared-libraries.html).
-Alternatively, the `LD_LIBRARY_PATH` environment variable can be set
-to the directory containing the `libappdynamics.so` file when
-starting {{site.base_gateway}}.
+library file is the only required file.
+
+### Recommended installation
+
+If you are using Kong version 3.0.0.0 or later, the recommended way to installed the `libappdynamics.so` is placed it in the `/usr/local/kong/lib` directory.
+This directory is included in the Kong search path for shared libraries, so the `libappdynamics.so` file will be found automatically.
```
Comment (cloudjumpercat) [inline app/_hub/kong-inc/app-dynamics/overview/_index.md:36]:
```suggestion
If you are using {{site.base_gateway}} 3.0.0.0 or later, we recommended installing the `libappdynamics.so` in the `/usr/local/kong/lib` directory.
This directory is included in the {{site.base_gateway}} search path for shared libraries, so the `libappdynamics.so` file will be found automatically.
```

File: app/_hub/kong-inc/app-dynamics/overview/_index.md [hunk 2]
```diff
@@ -28,12 +28,21 @@ not distributed with {{site.base_gateway}}. AppDynamics
...
+
+### Alternative installation
+
+If you are using an older version of Kong, or if you prefer to install the `libappdynamics.so` file in a different location, you can do so.
```
Comment (cloudjumpercat) [inline app/_hub/kong-inc/app-dynamics/overview/_index.md:40]:
```suggestion
If you are using an older version of {{site.base_gateway}}, or if you prefer to install the `libappdynamics.so` file in a different location, you can do so.
```

File: app/_hub/kong-inc/app-dynamics/overview/_index.md [hunk 3]
```diff
@@ -28,12 +28,21 @@ not distributed with {{site.base_gateway}}. AppDynamics
...
+
+- If Kong is deployed in RHEL or CentOS, the `libappdynamics.so` file can be placed in the `/usr/lib64` directory, which is included in the default search path for shared libraries.
```
Comment (cloudjumpercat) [inline app/_hub/kong-inc/app-dynamics/overview/_index.md:42]:
```suggestion
- If {{site.base_gateway}} is deployed in RHEL or CentOS, the `libappdynamics.so` file can be in the `/usr/lib64` directory, which is included in the default search path for shared libraries.
```

File: app/_hub/kong-inc/app-dynamics/overview/_index.md [hunk 4]
```diff
@@ -28,12 +28,21 @@ not distributed with {{site.base_gateway}}. AppDynamics
...
+- If Kong is deployed in Debian or Ubuntu, the `libappdynamics.so` file can be placed in the `/usr/lib` directory, which is included in the default search path for shared libraries.
```
Comment (cloudjumpercat) [inline app/_hub/kong-inc/app-dynamics/overview/_index.md:43]:
```suggestion
- If {{site.base_gateway}} is deployed in Debian or Ubuntu, the `libappdynamics.so` file can be in the `/usr/lib` directory, which is included in the default search path for shared libraries.
```

**Comment by cloudjumpercat** [comment]
I think the smoke test is failing because this is a fork, so I'll just approve and merge this.

**Review by cloudjumpercat — APPROVED** [review]
LGTM! Thanks for making this update!

**Comment by tzssangglass** [comment]
many thanks @cloudjumpercat

**Pull request merged**